include/mbedtls/sha512.h

```diff
@@ -212,19 +218,28 @@ extern "C" {
 #endif
 
 /**
- * \brief          Output = SHA-512( input buffer )
+ * \brief          This function calculates SHA-512 
```

Comment:
Blocker: Please align to SHA-1 PR: `This function calculates the SHA-512 or SHA-384 checksum of a buffer.`

Reply:
fixed